pkg/stagemanager/stage_manager.go

```diff
@@ -567,3 +569,65 @@ func (stm *StageManager) RunAll() {
 	// stop working
 	stm.Stop()
 }
+
+// StopMosnProcess stops Mosn process via command line,
+// and it not support stop on windows.
+func (stm *StageManager) StopMosnProcess() (err error) {
+	// init
+	stm.runParamsParsedStage()
+	stm.runInitStage()
+
+	mosnConfig := stm.data.config
+
+	// reads mosn process pid from `mosn.pid` file.
+	var pid int
+	if pid, err = pf.GetPidFrom(mosnConfig.Pid); err != nil {
+		log.StartLogger.Errorf("[mosn stop] fail to get pid: %v", err)
+		time.Sleep(100 * time.Millisecond) // waiting logs output
+		return
+	}
+
+	// sends SIGTERM to mosn process, makes it graceful exit.
+	proc, err := os.FindProcess(pid)
+	// check if process is existing.
+	err = proc.Signal(syscall.Signal(0))
+	if err != nil {
+		log.StartLogger.Errorf("[mosn stop] process(%v) is not existing, err: %v", pid, err)
+		time.Sleep(100 * time.Millisecond) // waiting logs output
+		return
+	}
+
+	if runtime.GOOS == "windows" {
+		log.StartLogger.Errorf("[mosn stop] stop command is NOT support on windows")
+		time.Sleep(100 * time.Millisecond) // waiting logs output
+		return
+	}
+
+	log.StartLogger.Infof("[mosn stop] sending INT signal to process(%v)", pid)
+	proc.Signal(syscall.SIGINT)
```

Comment:
```suggestion
	err := proc.Signal(syscall.SIGINT)
```
need error handling.